packages/core/src/external/aws/lambda.ts

```diff
@@ -97,9 +100,12 @@ export function getLambdaResultPayload({
   failOnEmptyResponse?: boolean;
   log?: typeof console.log;
 }): string | undefined {
-  if (result.StatusCode !== 200) {
+  if (!result.StatusCode || result.StatusCode < 200 || result.StatusCode > 299) {
```

Comment:
culprit of the [issue at staging](https://metriport.slack.com/archives/C04T5Q9JHFX/p1719331694970869?thread_ts=1719331458.061259&cid=C04T5Q9JHFX), it was invoking OK, just the response code was 202